src/tx/builder/schema.ts

```diff
@@ -926,8 +1059,71 @@ export const TX_SERIALIZATION_SCHEMA = {
   [TX_TYPE.payingFor]: {
     1: PAYING_FOR_TX
   }
+} as const
+
+export type TxType = keyof typeof TX_SERIALIZATION_SCHEMA
+
+export interface TxTypeSchemas {
```

Comment:
would require less code to generate it based on `TX_SERIALIZATION_SCHEMA` or so
including dropping the similar definitions like:
```
interface TxAccountsTree extends BuildTxArgBySchema<typeof ACCOUNTS_TREE_TX> {}
```